amazon-ec2-utils.spec

```diff
@@ -75,6 +75,14 @@ rm -rf $RPM_BUILD_ROOT
 /etc/udev/rules.d/60-cdrom_id.rules
 
 %changelog
+* Thu Jan 18 2024 Keith Gable <gablk@amazon.com> - 2.2.0-1
```

Comment:
This uses the same? version as the previous change from Guillaume (June 23) or did that version never release?

Reply:
That version never released